packages/components/pie-modal/README.md

```diff
@@ -0,0 +1,39 @@
+<p align="center">
+  <img align="center" src="../../../readme_image.png" height="200" alt="">
+</p>
+
+<p align="center">
+  <a href="https://www.npmjs.com/@justeattakeaway/pie-modal">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-modal.svg">
+  </a>
+</p>
```

Comment:
Is the above intentional?

Reply:
This is adapted from [pie-button README file](https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-button/README.md)